conda_forge_tick/migrators.xsh

```diff
@@ -356,8 +358,10 @@ class Compiler(Migrator):
         return (conditional or not any(x in attrs.get('req', []) for x in self.compilers))
 
     def migrate(self, recipe_dir, attrs, **kwargs):
-        self.out = $(conda-smithy update-cb3 --recipe_directory @(recipe_dir))
         with indir(recipe_dir):
+            content, self.messages = update_cb3('meta.yaml', get_cfp_file_path()[0])
```

Comment:
Can you store `get_cfp_file_path()[0]` and reuse it? It is expensive because it checks that the file is up-to-date.

Reply:
Could it be a class or module level variable?

Reply:
Sorry didn't see the changes. The init works too.